src/include/daos_task.h

```diff
@@ -212,6 +213,12 @@ typedef struct {
 	daos_target_info_t	*info_list;
 } daos_pool_query_target_t;
 
+typedef struct {
```

Comment:
(style)  do not add new typedefs

Reply:
Will keep this new typedef, because this patch is consistent with the other examples already found in this file.